src/pages/index.astro

```diff
@@ -0,0 +1,65 @@
+---
```

Comment:
Another day, another redesign. I like this one a lot more than #11, so I'll close that PR.